sechub-notification/src/main/java/com/mercedesbenz/sechub/domain/notification/user/NewApiTokenRequestedUserNotificationService.java

```diff
@@ -32,8 +32,8 @@ public void notify(UserMessage userMessage) {
         String tokenExpireDateTime = serviceHelper.getApiTokenExpireDate();
 
         StringBuilder emailContent = new StringBuilder();
-        emailContent.append("You requested a new API token. The token was created for you and expires at ");
-        emailContent.append(tokenExpireDateTime + ".\n");
+        emailContent.append("You requested a new API token. The link to retrieve it expires at ");
```

Comment:
Did you create a test email with the text? Some mail clients don't wrap long text lines.
Suggestion:
```suggestion
        emailContent.append("You requested a new API token.\nThe link to retrieve it expires at ");
```